infra/modules/providers/azure/keyvault/main.tf

```diff
@@ -0,0 +1,30 @@
+module "azure-provider" {
+  source = "../provider"
+}
+
+data "azurerm_resource_group" "kv" {
+  name = "${var.resource_group_name}"
+}
+
+data "azurerm_client_config" "current" {}
```

Comment:
How does this get populated?

Reply:
This is automatically set by terraform to the service principal running the deployment https://www.terraform.io/docs/providers/azurerm/d/client_config.html